packages/rpc-spec-types/src/rpc-response.ts

```diff
@@ -2,10 +2,11 @@ interface IHasIdentifier {
     readonly id: number;
 }
 
-type RpcErrorResponse = Readonly<{
+type RpcErrorResponseData = Readonly<{
```

Comment:
Little awkward to have `RpcErrorResponseData#data`. Maybe `RpcErrorResponsePayload`?

Reply:
How about `ResponseBody`? Payload is more often used for requests and Data/Body are more commonly used for responses.

Reply:
Oh actually this is only for `RpcErrorResponse` since `RpcResponse` doesn't have a nested `data` attribute. In this case, yeah I think `RpcErrorResponsePayload` is fine. 👌